src/SUMMARY.md

```diff
@@ -97,6 +97,7 @@
     - [Subtyping and variance](subtyping.md)
     - [Trait and lifetime bounds](trait-bounds.md)
     - [Type coercions](type-coercions.md)
+    - [Divergence](divergence.md)
```

Comment:
We try to keep the number of top-level chapter contained.  Looking at it, perhaps most of what's here that can't be inlined on the pages for each expression would make sense appearing in the [Expressions](https://doc.rust-lang.org/1.90.0/reference/expressions.html) chapter (e.g., we talk about place and value expressions there -- the verbiage about when a place expression is diverging might make sense near that) and in the [Never type](https://doc.rust-lang.org/1.90.0/reference/types/never.html) chapter.

Reply:
> We try to keep the number of top-level chapters contained.

My mistake.  I misread the diff.  As you noted elsewhere, this is a subchapter of the type system chapter.